.buildkite/pipeline.yml

```diff
@@ -62,3 +62,13 @@ steps:
           - step: "int-test"
             allow_failure: true
     depends_on: "check"
+
+  - label: "Publish docker image"
+    key: "publish"
+    command: ".buildkite/scripts/build_image.sh"
+    if: build.env("BUILDKITE_PULL_REQUEST") != null   #the condition is temporarily changed to opposite to check and fix the functionality
+    agents:
+      provider: "gcp"
+    depends_on:
+      - step: "tests"
+        allow_failure: false
```

Comment:
Thank you @ramonbutter for the suggestion, unfortunately, the error 404 for me :(

Reply:
hey sorry @sharbuz, the permissions may have been to restrictive on that repo.
Here is another example that should work for you: https://github.com/elastic/elasticsearch-controller/pull/147

Just wanted to highlight, that nearly around 20 services in the cloud cp needed to implement a similar logic like this, 
and we packed it in a dedicated buildkite agent so that we don't have to replicate the steps like everywhere and keep consistence in our build process.
It would have come further with the support to build ARM images, which is a little bit of a todo for this repository.

Reply:
Thank you @ramonbutter, I'll try to get access to the provided you examples because in the second example the same error 404 :(
CC @alexsapran , @mrodm could you help with the access or example?